docs/topics/mpp/multiplatform-library.md

```diff
@@ -7,7 +7,7 @@ This library converts raw data – strings and byte arrays – to the [Base64](h
 You will use different ways to implement the conversion to the Base64 format on different platforms:
 
 * For JVM – the [`java.util.Base64` class](https://docs.oracle.com/javase/8/docs/api/java/util/Base64.html).
-* For JS – the [Buffer API](https://nodejs.org/docs/latest/api/buffer.html).
+* For JS – the [`WindowOrWorkerGlobalScope.btoa()` API](https://developer.mozilla.org/docs/Web/API/WindowOrWorkerGlobalScope/btoa).
```

Comment:
Since the function is available on the global object (that's essentially what `WindowOrWorkerGlobalScope` expresses), I'd prefer we just call it the "btoa() function"